local_costmap:
  global_frame: /map
  robot_base_frame: base_link
  update_frequency: 6.0
  publish_frequency: 3.0
  static_map: false
  inflation_radius: 0.8
  rolling_window: true
  width: 5.0
  height: 4.0
  resolution: 0.05
  plugins:
   - {name: footprint_layer, type: "costmap_2d::FootprintLayer"}
   - {name: obstacle_layer, type: "costmap_2d::ObstacleLayer"}
   - {name: inflation_layer, type: "costmap_2d::InflationLayer"}
  obstacle_layer:
    observation_sources: laser
    laser: {topic: scan, data_type: LaserScan, expected_update_rate: 0.6, observation_persistence: 0.0, marking: true, clearing: true}
